library/src/main/java/com/datatorrent/lib/math/RunningAverage.java

```diff
@@ -64,9 +64,9 @@
     @Override
     public void process(Number tuple)
     {
-      double sum = (count * average) + tuple.doubleValue();
-      count++;
-      average = sum / count;
+      double sum = (RunningAverage.this.count * average) + tuple.doubleValue();
```

Comment:
@vrozov I created: https://issues.apache.org/jira/browse/APEXCORE-722
While making changes, I see that the change would break semantic versioning, as DefaultInputPort is not Evolving.